### PR TITLE
Allow setting terminal kernel parameters in saltboot formula

### PR DESCRIPTION
**Pull request by nadvornik**
https://github.com/SUSE/spacewalk/issues/12282

Kernel parameters specified in Saltboot formula are appended to kernel commandline for PXE, kexec and grub.

Kernel parameters specified in PXE formula are used only for PXE boot.

### CODE REVIEW

**Review by aaannz — APPROVED** [review]
Ok, but I do not feel very good by the functionality creep in fstab_updated. I think it should be split and stuff below 1643 could be in separate function. I mean what does fstab update have to do with setting kernel options?

**Comment by nadvornik** [comment]
You are right. I was thinking about renaming the function, but splitting new one is even better.

**Review by aaannz — APPROVED** [review]
One typo in comment, but looks good.

File: saltboot-formula/_states/saltboot.py [hunk 1]
```diff
@@ -1632,18 +1632,128 @@ def fstab_updated(name, partitioning, images):
             ret['comment'] += '\nfstab was not found.'
             return ret
 
+    return ret
+
+def bootloader_updated(name, partitioning, images, terminal_kernel_parameters=None):
+    '''
+    Ensure that both loacal bootloader and pxe server are updated
```
[the remark this answers is not in the document]
Comment (nadvornik) [reply saltboot-formula/_states/saltboot.py]: fixed

**Pull request merged**